website/markdown/docs/plugins/creating-plugins.mdx

```diff
@@ -0,0 +1,102 @@
+---
+name: Creating plugins
+excerpt: Learn how to create plugins for Tuist.
+---
+
+# Creating plugins
+
+## Plugin
+
+A plugin to Tuist is a directory with a `Plugin.swift` manifest. This manifest is used to define specific attributes of a plugin.
+
+## Plugin.swift manifest
+
+```swift
+import ProjectDescription
+
+let plugin = Plugin(name: "MyPlugin")
+```
+
+## Project description helpers
+
+Project description helpers can be used to extend Tuists functionality and Tuist automatically grabs the sources for
+these, compiles it, and allows it to be imported during project generation.
```

Comment:
```suggestion
Project description helpers can be used to extend Tuist's functionality: Tuist automatically compiles the sources and allows it to be imported during project generation.
```

I assume "it" here is the ProjectDescriptionHelper so might make sense to be more clear on that too